vignettes/MSstatsUpdated.Rmd

```diff
@@ -0,0 +1,324 @@
+
+This vignette summarizes the introduction and various options of all functionalities in MSstats. More details are available in `User Manual`.
+
+## __Installation__
+
+To install this package, start R (version “4.0”) and enter:
+
+``` {r code Installation}
+if (!requireNamespace("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+
+BiocManager::install("MSstats")
+library(MSstats)
+
+```
+
+## __1. Workflow__
+
+### __1.1 Raw Data__ 
+
+To begin with, we will load sample datasets, including both annotated and plain data. The dataset you need can be found [here](insert Link for dataset here).
+
+We will also load the Annotation Dataset using MSstatsConvert. You can access this dataset [here](insert Link for dataset here).
+
+
+``` {r code Load Dataset}
+library(MSstats)
+
+# Load data
+pd_raw = system.file("tinytest/raw_data/PD/pd_input.csv", 
+                    package = "MSstatsConvert")
+
+annotation_raw = system.file("tinytest/raw_data/PD/annot_pd.csv", 
+                   package = "MSstatsConvert")
+
+pd = data.table::fread(pd_raw)
+annotation = data.table::fread(annotation_raw)
+
+head(pd, 5)
+head(annotation, 5)
+
+```
+
+
+### __1.2 Converters__
+
+
+
+We have the following converters, which allow you to convert various types of output reports which include the feature level data to the required input format of MSstats. Further information about the converters can be found in the MSStatsConvert package.
+
+1. DIANNtoMSstatsFormat
+2. DIAUmpiretoMSstatsFormat
+3. FragPipetoMSstatsFormat
+4. MaxQtoMSstatsFormat
+5. OpenMStoMSstatsFormat
+6. OpenSWATHtoMSstatsFormat
+7. PDtoMSstatsFormat
+8. ProgenesistoMSstatsFormat
+9. SkylinetoMSstatsFormat
+10. SpectronauttoMSstatsFormat
+11. MetamorpheusToMSstatsFormat
+
+We show an example of how to use the above said Formatters. 
+
+```{r Converter Files}
+
+input_data = read.csv("../data/diann_report.tsv", 
+                      sep="\t")
+head(input_data)
+
+annotation_file = read.csv("../data/diann_annotation.csv")
+head(annotation_file)
+
+```
+
+```{r DIANNtoMSstatsFormat, results='hide', message=FALSE, warning=FALSE}
+
+msstats_format = MSstatsConvert::DIANNtoMSstatsFormat(input_data, 
+                                      annotation=annotation_file,
+                                      global_qvalue_cutoff = 0.01,
+                                      qvalue_cutoff = 0.01,
+                                      pg_qvalue_cutoff = 0.01,
+                                      useUniquePeptide = TRUE,
+                                      removeFewMeasurements = TRUE,
+                                      removeOxidationMpeptides = TRUE,
+                                      removeProtein_with1Feature = TRUE,
+                                      MBR=TRUE)
+```
+
+```{r DIANNtoMSstatsFormat head}
+head(msstats_format)
+```
+
+
+
+### __1.3 Loading PD Data to MSstats__
```

Comment:
Let's remove these lines since you already perform conversion with the PDtoMSstatsConverter